src/pandapipes/topology/topology_toolbox.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2020-2024 by Fraunhofer Institute for Energy Economics
+# and Energy System Technology (IEE), Kassel, and University of Kassel. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+from pandapipes.component_models.abstract_models.branch_models import BranchComponent
+import logging
+
+
+def get_all_branch_component_models():
+    """
+    Get all models of available branch components
+
+    :return: branch model
+    :rtype: list
+    """
+    def get_all_subclasses(cls):
+        all_subclasses = list()
+        for subclass in cls.__subclasses__():
```

Comment:
Does `__subclasses__` search for appearances in the whole python path? If not, we might need to find a different implementation that checks the appearing classes inside the net...

Reply:
Yes, __subclasses__ includes all child classes and is not dependent from net